packages/core/src/api/events/types.ts

```diff
@@ -0,0 +1,16 @@
+import type { TraceContext } from '../traces';
+
+export type EventAttributes = Record<string, string>;
```

Comment:
Is `string` the only acceptable value for an attribute?

Reply:
I figure it's better to be strict. Anyhow these will end up in logs as strings